src/App.jsx

```diff
@@ -14,33 +16,77 @@
   'Garlic',
 ];
 
-export const App = () => (
-  <div className="section content">
-    <div className="buttons">
-      <button type="button" className="button is-info is-light">
-        Sort alphabetically
-      </button>
+export const App = () => {
+  const [sortField, setSortField] = useState('');
+  const [reversed, setReversed] = useState(false);
+  const reset = () => {
+    setSortField('reset');
```

Comment:
Setting `sortField` to 'reset' is not handled in the sorting logic. Consider setting it to an empty string or a default value that is handled in the switch statement.